gas/shift/Sidebar.html

```
@@ -0,0 +1,109 @@
+    <div class="container">
+      <div class="label">選択中のセル範囲</div>
+      <p id="selected-range">:</p>
+      <button id="set-swapping-range" onclick="setSwappingRange()">選択中のセル範囲を登録する</button>
+      <hr>
+      <div class="label">登録中のセル範囲</div>
+      <p id="swapping-range">:</p>
+      <button id="swapping-values" onclick="swappingValues()">選択中のセル範囲と値を入れ替える</button>
+      <input type="checkbox" id="is-visualize-swap-area">
+    </div>
+    <div class="label">
+      <使い方><br>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)